src/ajax.js

```diff
@@ -13,6 +13,7 @@ define([
 
 var
 	rhash = /#.*$/,
+	rextraQuestions = /\?+/,
```

Comment:
@jaubourg oh.. ok.
I read these tests: https://github.com/jquery/jquery/blob/master/test/unit/ajax.js#L704
Finally, I don't understand exactly what is it necessary for doing some `jsonp`...
The doc http://api.jquery.com/jquery.ajax/ doesn't speak about `??` anywhere :(